### PR TITLE
Add tests for Format-Table -Wrap

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

Add new tests for `Format-Table -Wrap`.
Using '*' as placeholder because we use '...' eclipse If a value is wider than column width.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [ ] Not Applicable or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 [hunk 1]
```diff
@@ -465,4 +465,172 @@ er
...
+Long*********r3
+Head
+er
+----*-------*-----
+1**********2***3
+
+
+
+"@ },
+@{ view = "Default"; widths = 4,7,5; variation = "narrow values with wrap"; values = [PSCustomObject]@{First=1;Second=2;Third=3}; wrap = $true; expectedTable = @"
```
Comment (SteveL-MSFT) [inline test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: Seems like inconsistent indenting of the test cases.  My preference is to have each new test case align with line 470.

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 [hunk 2]
```diff
@@ -465,4 +465,172 @@ er
...
+"@ },
+@{ view = "Default"; widths = 4,7,5; variation = "wide values with wrap"; values = [PSCustomObject]@{First="12345";Second="12345678";Third="123456"}; wrap = $true; expectedTable = @"
+
+Long*Header2*Heade
+Long*********r3
+Head
+er
+----*-------*-----
+1234*1234567*12345
+5**********8*6
```
Comment (SteveL-MSFT) [inline test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: Do we need a test case where not all the values wrap?  Maybe have 1st and 3rd wrap, and the another where only 1st wraps.

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]
Thanks for adding these test variations!

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 (hunk 1 above)
Comment (iSazonov) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: Fixed.

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1 (hunk 2 above)
Comment (iSazonov) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1]: I agree.
Added.

**Review by SteveL-MSFT — APPROVED** [review]

**Pull request merged**